elife/config/etc-updatedb.conf

```diff
@@ -0,0 +1,4 @@
+PRUNE_BIND_MOUNTS="yes"
+# PRUNENAMES=".git .bzr .hg .svn"
+PRUNEPATHS="/tmp /var/spool /media /home/.ecryptfs /ext"
```

Comment:
do we use encryption anywhere? Well, no harm

Reply:
no, but I think builder-private is a real concern. I think new version of Salt allows you to mount an encrypted pillar on the master and target values within it